app/liquid_tags/liquid_tag_base.rb

```diff
@@ -51,6 +51,10 @@ def pattern_match_for(input, regex_options)
       .first
   end
 
+  def render_tag(klass, tag_name, link, parse_context)
+    klass.__send__(:new, tag_name, link, parse_context).render(nil)
+  end
+
```

Comment:
Looks like this is no longer needed with the `.new` change 🙂 

```suggestion
```

Reply:
Did I not delete this? 🤦🏾‍♀️